File: leetcode-solutions/2357.py
from typing import List

class Solution:
    def minimumOperations(self, nums: List[int]) -> int:
        li = set()
        for num in nums:
            if num != 0:
                li.add(num)

        return len(li)
    


nums = [1,5,0,3,5]
print(Solution().minimumOperations(nums))

File: leetcode-solutions/2357.js
function minimumOperations(nums) {
    var result = [];
    for (var _i = 0, nums_1 = nums; _i < nums_1.length; _i++) {
        var num = nums_1[_i];
        if (num !== 0 && !(num in result)) {
            result.push(num);
        }
    }
    return result.length;
}
;
var nums = [];
var result = minimumOperations(nums);
console.log(result);
